be/src/storage/lake/tablet_manager.h

```diff
@@ -70,6 +70,11 @@ class TabletManager {
     StatusOr<TabletMetadataPtr> publish_version(int64_t tablet_id, int64_t base_version, int64_t new_version,
                                                 const int64_t* txns, int txns_size);
 
+    // Used in UT now, will not clear trash files after publish finish
+    StatusOr<TabletMetadataPtr> publish_version_without_clear_files(int64_t tablet_id, int64_t base_version,
```

Comment:
Why need this method in UT

Reply:
Because if publish finish, it will clear `TxnLog` by default. I want to check `TxnLog` in meta cache after publish finish, so I add this method.

Reply:
However, this would make the behavior in testing inconsistent with actual behavior. I think we should keep the tests as close to real runtime behavior as possible.